lib/Java2ClientGenerator.php

```diff
@@ -500,6 +500,12 @@ public function getPropertyValue($propName, $propType, $propertyNode) {
 				$primitiveType = "double";
 				break;
 			case "int":
+				$isTime = $propertyNode->hasAttribute("isTime");
```

Comment:
1. the cases for int and string had the same handling before your change, not int is not handled.
2. would be better to move $isTime = $propertyNode->hasAttribute("isTime"); before the if and if its true assign time as the prop type instead of adding it in the int case